templates/csharp/xplat/AvaloniaTest.Browser/AvaloniaTest.Browser.csproj

```diff
@@ -1,16 +1,9 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk.WebAssembly">
```

Comment:
Microsoft.NET.Sdk.WebAssembly is only supported with 11.1, so for 11.0 we will need to keep old csproj format

Reply:
Optional: add a parameter like "Use Microsoft.NET.Sdk.WebAssembly" to allow old behavior for 11.1 as well